packages/fast-tooling-react/src/form/controls/control.children.tsx

```diff
@@ -39,6 +39,19 @@ class ChildrenControl extends React.Component<
         managedClasses: {},
     };
 
+    public static getDerivedStateFromProps(
+        props: ChildrenControlProps,
+        state: ChildrenControlState
```

Comment:
You're doing somthing very similar in `compnentDidUpdate` - is that still needed?

Reply:
Turns out it was unnecessary, removed.